src/dxtbx/model/beam.h

```diff
@@ -403,6 +405,169 @@ namespace dxtbx { namespace model {
     return os;
   }
 
+  class PolyBeam : public Beam {
+  public:
+    PolyBeam() {
+      set_direction(vec3<double>(0.0, 0.0, 1.0));
+      set_divergence(0.0);
+      set_sigma_divergence(0.0);
+      set_polarization_normal(vec3<double>(0.0, 1.0, 0.0));
+      set_polarization_fraction(0.999);
```

Comment:
This reminds me, this is the default suitable for synchrotron X-ray sources. I guess we might keep that default for consistency, but it would not be right for neutron or electron beams. An unpolarised beam has (a bit confusingly) a `polarization_fraction=0.5`.